.gitignore

```diff
@@ -10,7 +10,6 @@ Pipfile*
 # Django
 db.sqlite3
 media/
-static/
```

Comment:
This isn't addressed.  By doing this, the root-level static folder shows as untracked instead of ignored.  It should be ignored.  Use appropriate regex's to do what you mean and not what you said.